lib/service_actor/type_checkable.rb

```diff
@@ -31,7 +31,7 @@ def self.included(base)
 
   module PrependedMethods
     DEFAULT_MESSAGE = lambda do
-      |kind, input_key, service_name, expected_type_names, actual_type_name|
+      |kind:, input_key:, service_name:, expected_type_names:, actual_type_name:| # rubocop:disable Layout/LineLength
 
       "#{kind} #{input_key} on #{service_name} must be of type " \
       "#{expected_type_names} but was #{actual_type_name}"
```

Comment:
Since we are now exposing these, I would be inclined to make sure the naming is the same for all arguments.

Here for example `kind` is the same as `NilCheckable`’s `origin`… but capitalized.

We could add a little logic here to capitalize an `origin` argument (e.g. `kind = origin == "input" ? "Input" : "Output"`). Or we can change the wording altogether so as not to have is a first word, which would be much simpler.

Naming-wise I think I’d also like to rename `expected_type_names` to `expected_type`. Conceptually I think I’d describe `[User, Admin]` as being **one** type that matches several classes. Also I would rename `actual_type_name` to `given_type` as I find it a bit clearer.

All in all this could look like this:

```suggestion
      |input_key:, origin:, expected_type:, given_type:|
      "The #{input_key} #{origin} must be of type " \
      "#{expected_type} but was #{given_type}"
```

Reply:
I renamed it to `expected_type` and to `given_type`. I like these names.

Reply:
I brought the beginning of all default texts to more or less the same form. Next, I will need to add the same double quotes to all names inserted into the text.